README.md

```diff
@@ -0,0 +1,64 @@
+# LinkedList
+
+[![Crates.io](https://img.shields.io/crates/v/linked_list_r4l)](https://crates.io/crates/linked_list_r4l)
+[![Doc.rs](https://docs.rs/linked_list_r4l/badge.svg)](https://docs.rs/linked_list_r4l)
+[![CI](https://github.com/arceos-org/linked_list/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/arceos-org/linked_list/actions/workflows/ci.yml)
```

Comment:
Rename this repo to `linked_list_r4l`

Reply:
Ok, but renaming the repository requires you to complete this operation